hail_scripts/reference_data/config.py

```diff
@@ -68,7 +68,7 @@ def custom_gnomad_select_v3(ht):
     },
 """
 CONFIG = {
-    '1kg': {  # tgp
+    'tgp': {  # tgp
```

Comment:
I'm pretty sure we don't use this anymore at all in seqr. @lynnpais just confirming that we do not want to show Thousand Genomes frequencies in seqr anymore?

Reply:
TGP is within gnomAD v3.1 frequencies if that helps make a decision.

Reply:
Confirming we do no longer want to show or search on thousand genomes frequencies. Thanks.

Reply:
okay so we can remove this entirely!